contributing/pull_request.md

```diff
@@ -98,6 +98,10 @@ class/interface/trait, method, and variable.
 
 Do not add PHPDoc comments that are superficial, duplicated, or stating the obvious.
 
+It is not recommended to reuse comments if the parent class or interface already contains a description of the child element.
```

Comment:
I think this can be combined with line 99 above. and the pull request checklist needs to be updated too